catkin_ws/src/sim/src/unity_bridge.py

```diff
@@ -61,61 +59,59 @@ def cb_unity_state(msg):
 
     # IMU - NED
     if isIMUActive:
+        pub_imu_sensor_status.publish(Bool(True))
         sbg_quat_msg = SbgEkfQuat()
-        sbg_quat_msg.quaternion = Quaternion(pose_quat_x, pose_quat_y, pose_quat_z, pose_quat_w)
+        q_NED_imunominal = np.quaternion(q_NED_imunominal_w, q_NED_imunominal_x, q_NED_imunominal_y, q_NED_imunominal_z)
+
+        q_NED_imu = q_NED_imunominal * q_imunominal_imu
+        sbg_quat_msg.quaternion = Quaternion(q_NED_imu.x, q_NED_imu.y, q_NED_imu.z, q_NED_imu.w)
         pub_imu_quat_sensor.publish(sbg_quat_msg)
 
         sbg_data_msg = SbgImuData()
-        sbg_data_msg.gyro = Vector3(twist_angular_x[0], twist_angular_y[1], twist_angular_z[2])
+        sbg_data_msg.gyro = Vector3(twist_angular_x, twist_angular_y, twist_angular_z)
```

Comment:
unity gives angular velocity in a global reference frame not a local one. A rotation is necessary to make this correct. Look at how it is done in the bypass bridge

Reply:
Yeah makes sense